gateway/templates/istio-ingress/vsvc.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.istioIngress.enabled }}
+{{- if .Values.gatewayModes.outboundDlp.enabled }}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: gateway
+spec:
+  gateways:
+    - {{ include "gateway.name" . }}-outbound-dlp
+  hosts:
+    - '*' # Should be changed with the hostname of the mail server.
```

Comment:
Is this something that is known ahead of time? If, so lets template it as a value

Reply:
I believe the Istio Gateway should be the FQDN of the domain, and the destination host should be the name of the service.